python/ray/scripts/scripts.py

```diff
@@ -445,6 +445,32 @@ def attach(cluster_config_file, start, cluster_name):
     attach_cluster(cluster_config_file, start, cluster_name)
 
 
+@click.command()
+@click.argument("cluster_config_file", required=True, type=str)
+@click.argument("source", required=True, type=str)
+@click.argument("target", required=True, type=str)
+@click.option(
+    "--cluster-name",
+    required=False,
+    type=str,
+    help=("Override the configured cluster name."))
+def rsync_down(cluster_config_file, source, target, cluster_name):
```

Comment:
Can you add some descriptors for these functions?

Reply:
That's not currently done in this file. Do those show up in ray --help?